.github/dependabot.yml

```diff
@@ -0,0 +1,283 @@
+version: 2
+registries:
+  visual-studio-securitytools:
+    type: nuget-feed
+    url: https://securitytools.pkgs.visualstudio.com/_packaging/Guardian/nuget/v3/index.json
+    # token: ${{ secrets.NUGET_PRIVATE_REG_TOKEN }}
```

Comment:
To allow Dependabot suggest updates for dependencies in `eng/common/sdl/packages.config`:
- add registry token as GitHub secret with the name `NUGET_PRIVATE_REG_TOKEN` and uncomment